reporter/alert-prod.ts

```diff
@@ -0,0 +1,108 @@
+import { SuperfaceClient } from '@superfaceai/one-sdk';
+import { TestAnalysis, TestReport } from '@superfaceai/testing';
+
+const parseAnalysis = (analysis: TestAnalysis): any => {
+  const addedErrors = analysis.errors.added.join('\n');
+  const removedErrors = analysis.errors.removed.join('\n');
+  const changedErrors = analysis.errors.changed.join('\n');
+
+  const errorsCount = [
+    ...analysis.errors.added,
+    ...analysis.errors.removed,
+    ...analysis.errors.changed,
```

Comment:
Not entirely sure but this feels like not very effective way. I think you are allocating entire array and filling it just to get length. I would do just:
  const errorsCount = 
     analysis.errors.added.legth +
     analysis.errors.removed.legth +
     analysis.errors.changed.legth